modules/installation-nutanix-failure-domains-req.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/nutanix-failure-domains.adoc
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-nutanix-failure-domains-req_{context}"]
+= Failure domain requirements
+
+When planning to use failure domains, consider the following requirements:
+
```

Comment:
I assume all the references to Prism Elements mean the same thing, yet we use _Prism Element instances_, _Prism Element clusters_, and _each Prism Element_. I think we should be consistent and just use _Prism Element_. Should we use _Nutanix Prism Element (cluster)_ on the first reference?

Reply:
I believe references to Prism Element (cluster), Prism Element(s), and Prism Element instances are OK as is. I am comfortable moving between references and this will resonate with the intended audience. Agreed that on first instance, we should state Nutanix. I will update accordingly.